systemtests/src/main/java/io/enmasse/systemtest/messaginginfra/resources/NamespaceResourceType.java

```diff
@@ -20,7 +20,7 @@ public Namespace get(String namespace, String name) {
     }
 
     public static Namespace getDefault() {
-        return new NamespaceBuilder().withNewMetadata().withName("enmasse-app").endMetadata().build();
+        return new NamespaceBuilder().withNewMetadata().withName("enmasse-tenant").endMetadata().build();
```

Comment:
I wonder if this should just be a random UUID.

Reply:
again good point I will fix it